src/safeposix/syscalls/fs_calls.rs

```diff
@@ -1,4 +1,5 @@
 // File system related system calls
+use libc::{LOCK_NB,LOCK_UN,LOCK_SH,LOCK_EX};
```

Comment:
we shouldn't put any use's in here for consistency, even if these are just constants. let's just define these in our constants file?